### ISSUE TITLE
Hide React EU?

### ISSUE THREAD

**Issue by madewild** [ec-doris/kohesio-frontend#958]
https://kohesio.ec.europa.eu/en/projects?theme=React-EU is showing no results
@Dgojsic is it normal? Should we hide this theme on https://kohesio.ec.europa.eu/en/themes?

**Comment by Dgojsic** [comment]
No, it's not normal, probably the filter doesn't work. Projects are tagged as such, but don't show up when filtered for. See for example this project with ReactEU as a theme https://kohesio.ec.europa.eu/en/projects/Q4697437 .

**Comment by madewild** [comment]
OK so it's a backend issue, transferring
